events.go

```diff
@@ -133,6 +133,9 @@ func (s *Session) handleNodeEvent(frames []frame) {
 		// TODO: can we be sure the order of events in the buffer is correct?
 		switch f := frame.(type) {
 		case *topologyChangeEventFrame:
+			host, port := s.cfg.translateAddress(f.host.String(), f.port)
+			f.host = net.ParseIP(host)
```

Comment:
I think it's probably best to move the translation to the layer below event deduplication that way we dont need to do this in every event.